packages/ui-modal/src/Modal/props.ts

```diff
@@ -156,7 +149,7 @@ type ModalOwnProps = {
   /**
    * The children to be rendered within the `<Modal />`. Children must be type of: `Modal.Header`, `Modal.Body`, `Modal.Footer`. The `Modal.Body` child is required, and they have to follow this order.
```

Comment:
This is no longer true. Id rewrite this that these are the recommended children, but one can use custom children and that the `variant` and `overflow` props are always passed down